c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -106,6 +106,10 @@ replace your sencha touch interface by an ngeo interface.
       </script>
       % endif
 
+13. In the PDF report we now make a list id ids, then we should change:
+    - The `paramID` is replaced by a `datasource` with an `id` and it should be a `!datasource` with a `id: !string`.
```

Comment:
The way BS is using that feature is that they do not define a datasource in `config.yaml` but directly in the template.